lib/jake.ex

```diff
@@ -16,6 +18,17 @@ defmodule Jake do
     |> StreamData.one_of()
   end
 
+  def gen(%{"allOf" => options} = spec) when is_list(options) do
+    properties =
+      options
+      |> Enum.reduce(%{}, fn x, acc -> x |> Jake.MapUtil.deep_merge(acc) end)
```

Comment:
use pipe only if it will make the code more readable. Don't try to use it everywhere.

Reply:
we already have a alias for MaptUtil, no need to use full name